tests/override_config_table/test_override_config_table.py

```diff
@@ -0,0 +1,153 @@
+import json
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.config_reload import config_reload
+
+GOLDEN_CONFIG = "/etc/sonic/golden_config_db.json"
+GOLDEN_CONFIG_BACKUP = "/etc/sonic/golden_config_db.json_before_override"
+CONFIG_DB = "/etc/sonic/config_db.json"
+CONFIG_DB_BACKUP = "/etc/sonic/config_db.json_before_override"
+
+logger = logging.getLogger(__name__)
+
+
+def file_exists_on_dut(duthost, filename):
+    return duthost.stat(path=filename).get('stat', {}).get('exists', False)
+
+
+@pytest.fixture(scope="module")
+def golden_config_exists_on_dut(duthost):
+    return file_exists_on_dut(duthost, GOLDEN_CONFIG)
+
+
+def backup_config(duthost, config, config_backup):
+    logger.info("Backup {} to {} on {}".format(
+        config, config_backup, duthost.hostname))
+    duthost.shell("cp {} {}".format(config, config_backup))
+
+
+def restore_config(duthost, config, config_backup):
+    logger.info("Restore {} with {} on {}".format(
+        config, config_backup, duthost.hostname))
+    duthost.shell("mv {} {}".format(config_backup, config))
+
+
+def get_running_config(duthost):
+    return json.loads(duthost.shell("sonic-cfggen -d --print-data")['stdout'])
```

Comment:
>"sonic-cfggen -d --print-data")['stdout'] [](http://example.com/codeflow?start=36&length=41)

Is it better to use `show runningconfiguration all`?

Reply:
I think they are the same.